README.md

```diff
@@ -1,2 +1,16 @@
 # OpenAds-ConnectorAPI
-Complaint interfaces to be implemented to develop an OpenAds connector
+
+Compliant interfaces to be implemented to develop an OpenAds connector.
```

Comment:
I would use: 

_Compliant interfaces to be implemented when developing an OpenAds connector._

Reply:
It was already fixed 👍